doc/source/conf.py

```diff
@@ -147,13 +149,9 @@ def setup(app):
 
 autosectionlabel_prefix_document = True
 
-try:
-    import sphinx_autodoc_typehints
-
+if find_spec("sphinx_autodoc_typehints") is not None:
```

Comment:
Cool?  I thought try/import/except was ok, or was it complaining about the pass?  We'd usually do a vollog there to say that the import wasn't found but also wasn't critical...

Reply:
it was complaining about an unused import :)

Reply:
Oh, I think a fix for that is to include `import thing as thing` and then it won't complain?  I think I'd prefer that than this which is just a little more cognitive load for people that aren't familiar with it...

Reply:
you'd still have an unused import tho

Reply:
Yeah, ok, it's not a showstopper and I guess we'll get used to it...  5;P